server/controller/planRoutes.js

```diff
@@ -6,6 +6,7 @@ const verifier = require('../shared/verifyPermissions');
 
 // All the routes for plans
 router.get('/get', middleware.checkToken, planHandler.getPlans);
+router.post('/get', middleware.checkToken, planHandler.getPlan);
```

Comment:
How would we otherwise retrieve one specific row from a table, though? I believe that post is the only request type that allows you to pass and receive a body during a request.

Reply:
Interesting. You are correct! You learn something new every day. However, we are still performing a get operation, so post still sounds bad. My new recommendation: we bring in dynamic routing to handle this. The route would therefore be `router.get('/get/:e_id')`, and using dynamic routing on the server side, it'd be easy as pie to get the id from the address without having to attach a body.

Reply:
That sounds like a good idea. It does seem strange to be making a get operation using a post request. I'll include the updated routes/handlers in my updated pull request.